### ISSUE TITLE
Doesn't keep the server alive

### ISSUE THREAD

**Issue by cburgdorf** [nickel-org/nickel.rs#188]
This doesn't keep the server running.

```rust
#[macro_use] extern crate nickel;

use nickel::Nickel;
use nickel::router::http_router::HttpRouter;

fn main() {
    let mut server = Nickel::new();

    server.get("/bar", middleware!("foo"));
    server.listen("127.0.0.1.6767");
}
```

**Comment by Ryman** [comment]
Should be `127.0.0.1:6767` ?

**Comment by cburgdorf** [comment]
Oooops, how is that not a compile time error?

**Comment by Ryman** [comment]
It should be a runtime error? http://is.gd/ynNsUe

**Comment by Ryman** [comment]
Fixed in #198
